data/multiplayer/scenarios/4p_King_of_the_Hill.cfg

```diff
@@ -1,10 +1,38 @@
 #textdomain wesnoth-multiplayer
 
+#define SPAWN_HEIR SIDE
+    [store_side]
+        side={SIDE}
+    [/store_side]
+    {RANDOM $side.recruit}
+    [store_unit_type]
+        type=$random
+    [/store_unit_type]
+    # If the unit is level 0 then switch to one of its advancements instead.
+    [if]
+        [variable]
+            name=unit_type.level
+            equals=0
+        [/variable]
+        [then]
+            {RANDOM $unit_type.advances_to}
+        [/then]
+    [/if]
+    [unit]
```

Comment:
Should this guy use the "disposable leader" overlay? Or do you get a game over when he dies?

Reply:
You get a game over only when you have no leaders left.

Reply:
So in that case it kind makes sense to call _both_ your leaders disposable when you have two, but when you have one, they're not disposable…

Probably not worth doing anything special in that case, I think.

Reply:
Makes sense; I will leave the default leadership overlay in place then.